scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -129,37 +129,49 @@ case object RedundantBraces extends Rewrite {
         implicit val builder = Seq.newBuilder[TokenPatch]
         builder += TokenPatch.Remove(lbrace)
         builder += TokenPatch.Remove(rbrace)
-        removeTrailingLF(lbrace.pos, rbrace)
-        ctx.tokenTraverser
-          .reverseFind(rbrace)(!Whitespace.unapply(_))
-          .foreach(t => removeTrailingLF(t.pos, rbrace))
+        removeLFToAvoidEmptyLine(rbrace)
         ctx.addPatchSet(builder.result(): _*)
       }
     case _ =>
   }
 
-  private def removeTrailingLF(
-      bodyEnd: Position,
-      close: Token
-  )(implicit builder: PatchBuilder, ctx: RewriteCtx): Unit =
-    if (close.pos.startLine != bodyEnd.endLine) {
-      import ctx.tokenTraverser._
-      val next = nextToken(close)
-      if (next.is[LF])
-        builder += TokenPatch.Remove(next)
+  private def isLFSkipWhitespace(token: Token): Option[Boolean] =
```

Comment:
Can we add brief scaladoc that explains that this method is used with the `findAfter/findBefore`? because as long as this method isn't passed to the predicate of `findAfter/Before`, this method doesn't make sense, and it's a bit confusing for me.